src/route-handlers/list-workflows/helpers/__tests__/get-list-workflow-executions-query.test.ts

```diff
@@ -18,7 +18,7 @@ describe('getListWorkflowExecutionsQuery', () => {
   it('should return query for running status', () => {
     const query = getListWorkflowExecutionsQuery({
       search: 'mocksearchterm',
-      workflowStatus: 'WORKFLOW_EXECUTION_STATUS_RUNNING',
+      workflowStatus: 'WORKFLOW_EXECUTION_CLOSE_STATUS_INVALID',
```

Comment:
hmmm if you check workflow_status, for running workflow it should be WORKFLOW_EXECUTION_STATUS_RUNNING.

From proto definitions
```
enum WorkflowExecutionStatus {
    WORKFLOW_EXECUTION_STATUS_UNSPECIFIED = 0;
    // Value 1 is hardcoded in SQL persistence.
    WORKFLOW_EXECUTION_STATUS_RUNNING = 1;
    WORKFLOW_EXECUTION_STATUS_COMPLETED = 2;
    WORKFLOW_EXECUTION_STATUS_FAILED = 3;
    WORKFLOW_EXECUTION_STATUS_CANCELED = 4;
    WORKFLOW_EXECUTION_STATUS_TERMINATED = 5;
    WORKFLOW_EXECUTION_STATUS_CONTINUED_AS_NEW = 6;
    WORKFLOW_EXECUTION_STATUS_TIMED_OUT = 7;
}
```